src/components/Controls/Controls.css

```diff
@@ -0,0 +1,16 @@
+.controls {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  padding: 2rem;
```

Comment:
generally ```rem``` is better for font-size rather than margin/padding — if a user has a larger default text size, ```rem``` is good because it will scale up the font-size based on that default (they want bigger text so they can read it easier) but they probably don't need bigger margins/padding so try to stick to ```px```